README.md

```diff
@@ -0,0 +1,39 @@
+# Books API
+
+## Database scheme 
+```mermaid
+erDiagram
+Book{
+  long id
+  string title
+  string annotation
+  datetime createdAtUtc
+  string artworkUrl
+  int numberOfCopies
```

Comment:
I suggest removing this field, because we can count the number of books using COUNT from sql if we need it. Without this thing, each record in the database will be a single instance of a book, with such a clear how to assign the status “On hands”/“In office” without complex logic. One row - one book.

Reply:
I think it will be necessary to coordinate this with Vera. In any case, it seems to me that even if on the frontend it ends up that we can leave the number of books new in the field right away, and there will be 2 of them, on the backend we can still create two records about it in the database. I think this field might be a complication, because then it's still not clear what is meant by this field - the total number of these books or the number of these books in the office at the moment. If the second - then we will not be able to understand at any time how many such books we have at all, this database will only give an understanding of how many such books are in the office at the moment. There is a probability of simply losing a duplicate book in such a flow.